scoping/api/scoping.api

```diff
@@ -119,8 +119,6 @@ public final class com/dropbox/kaiken/scoping/BuildConfig {
 	public static final field BUILD_TYPE Ljava/lang/String;
 	public static final field DEBUG Z
 	public static final field LIBRARY_PACKAGE_NAME Ljava/lang/String;
-	public static final field VERSION_CODE I
-	public static final field VERSION_NAME Ljava/lang/String;
```

Comment:
@changusmc I had to remove these. That was a change with AGP 4.1, see https://issuetracker.google.com/issues/154275579#comment3 Please take another look.

Reply:
If you prefer, then I can bring back these values for each module. But to me it seems unnecessary.

Reply:
I think it's okay, I doubt people are using these APIs